### DISCUSSION TITLE
How to get user input?

### DISCUSSION THREAD

**Discussion by brayner05** [gleam-lang/gleam#2748] [Questions & support] [answered]
I am trying to read a line from the user:
```
import gleam/io


pub fn main() {
    io.print("> ")
    let line = ???
    io.println(line)
    main()
}
```
but I can't find any documentation on reading user input.

**Comment by lpil** [comment, marked as answer]
This function in the gleam_erlang package may be what you're after

https://hexdocs.pm/gleam_erlang/gleam/erlang.html#get_line

**Reply by brayner05** [reply]
Yep! Thanks!
